ci/helm/helmfile.yaml

```diff
@@ -15,27 +17,44 @@ releases:
     chart: nuxeo/nuxeo
     version: 3.0.11
     needs:
-      - elasticsearch
-      - mongodb
+      - {{ eq .Environment.Name "functional-tests-2023" | ternary "opensearch" "elasticsearch" }}
+      - {{ eq .Environment.Name "functional-tests-2023" | ternary "mongodb6" "mongodb4" }}
```

Comment:
We talked about overriding the host in mongo values but I didn't find the right value yet. However. This condition is also done just above for ES / OS just above so we only have one value to change.